vendor/github.com/mongodb/go-client-mongodb-atlas/mongodbatlas/maintenance.go

```diff
@@ -0,0 +1,136 @@
+package mongodbatlas
```

Comment:
Skipping reviewing maintenance related as that's in a separate PR.  Please ensure each PR just contains changes related to task at hand please.

Reply:
sorry I will remove it

Reply:
Thank you!

Reply:
I can't remove because is the vendor files that are dependencies

Reply:
Ah, that's fine.  At least we know it's not been reviewed yet.

Reply:
Yes, for this reason, exists the vendor files, sorry for not explain you this before